client/components/chat-list-section.jsx

```diff
@@ -21,7 +21,30 @@ export default class ChatListSection extends React.Component {
   }
 
   componentDidMount() {
-    this.getAllChatRooms();
+    fetch('/api/chatRooms')
+      .then(response => response.json())
+      .then(result => {
+        const usersRooms = [];
+        for (let i = 0; i < result.length; i++) {
+          if (this.state.user.chatRooms.includes(result[i].id)) {
+            usersRooms.push(result[i]);
+          }
+        }
```

Comment:
When you want to conditionally include values in a new array, then reach for `.filter()`

```js
const usersRooms = result.filter(room => this.props.user.chatRooms.includes(room.id))
```